EIPS/eip-7002.md

```diff
@@ -73,11 +73,9 @@ Note that `amount` is returned by the contract little-endian, and must be encode
 
```

Comment:
> The [EIP-7685](./eip-7685) encoding of a withdrawal request is computed as follows.

Can you also fix the above link to `[EIP-7685](./eip-7685.md)`? It is missing the md extension

Reply:
fixed. thx